README.md

```diff
@@ -29,34 +29,7 @@ If you would like to allow other apps to play music over your video component, a
 
 #### Android
 
-First, copy your video file to `android/app/src/main/res/raw/`, then
-make the following additions to the given files:
-
-**android/settings.gradle**
-```
-include ':RCTVideo', ':app'
-project(':RCTVideo').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android')
-```
-
-**android/app/build.gradle**
-```
-dependencies {
-   ...
-   compile project(':RCTVideo')
-}
-```
-
-**MainActivity.java**
-
-On top, where imports are:
-```java
-import com.brentvatne.react.ReactVideoPackage;
-```
-
-Under `.addPackage(new MainReactPackage())`:
-```java
-.addPackage(new ReactVideoPackage())
-```
+Install [rnpm](https://github.com/rnpm/rnpm) and run `rnpm link react-native-video`
```

Comment:
Just a suggestion, but I think both rnpm & the manual install methods should be listed. I've noticed that rnpm fails sporadically, so I've taken to manually installing all packages. I've talked to a few other devs using RN and they're apparently doing the same.